lang/spec.html

```diff
@@ -8233,15 +8233,15 @@ <h3>Worker message passing</h3>
 <li>a send failure type: this is the type of termination failure in the sending
 worker that caused the corresponding message not to be sent; more precisely, for
 any slot index <var>i</var>, if it is possible for <var>i</var> to be &#x2265;
-the value of index <var>w</var> of the queue at the time when the sending worker
+the index <var>w</var> of the queue at the time when the sending worker
```

Comment:
When we say "the value of index i", one could confuse it with the slot's value at index i.